user_sync/rules.py

```diff
@@ -204,12 +204,8 @@ def validate_and_log_additional_groups(self, umapi_info):
         """
         umapi_name = umapi_info.get_name()
         for mapped, src_groups in umapi_info.get_additional_group_map().items():
-            if len(src_groups) > 1:
```

Comment:
The additional groups should map 1:1 to target groups.  We need to enforce that with this check.

Reply:
I think the idea was to allow this kind of mapping.  I can't say I see a lot of use cases, but it does seem like it would be convenient to be able to combine source groups into a target group if desired.

```python

   # In the first case, we map anything that looks like (umapi name) Group (any)
  # into a group called Externally Mapped on each of the secondary umapies
  # this will throw if the validation is in place -- but perhaps this is intended

    - source: "([^\\s]+) Group (.+)"
      target: "\\1::Externally Mapped"

   # A simpler scenario: Map all the groups that look like Group (XYZ) into a 
   # pooled group on the target console (no secondaries involved).  This simpler
   # case will also fail the validation check - do we want to intentionally block this
   # kind of mapping?

    - source: "Group (.+)"
      target: "Externally Mapped"
````